modules/tinymce/src/themes/silver/test/ts/browser/editor/toolbar/ToolbarDrawerToggleTest.ts

```diff
@@ -68,4 +69,43 @@ describe('browser.tinymce.themes.silver.editor.toolbar.ToolbarDrawerToggleTest',
       pTestToggle({ toolbar: [ 'undo redo', 'bold italic' ], inline: true }, false)
     );
   });
+
+  const pTestEvent = async (toolbarMode: ToolbarMode, command: (editor: Editor) => void) => {
+    const editor = await McEditor.pFromSettings<Editor>({
+      menubar: false,
+      statusbar: false,
+      width: 200,
+      toolbar_mode: toolbarMode,
+      base_url: '/project/tinymce/js/tinymce'
+    });
+    editor.focus();
+
+    const lastState = Singleton.value<boolean>();
+    lastState.set(false);
+    await UiUtils.pWaitForEditorToRender();
+
+    editor.on('ToggleToolbarDrawer', (options: { isToolbarDrawerToggled: boolean }) => {
+      lastState.set(options.isToolbarDrawerToggled);
+    });
+
+    assert.isFalse(lastState.get().getOrDie('Should be set'));
+    command(editor);
+    await Waiter.pWait(10);
+    assert.isTrue(lastState.get().getOrDie('Should be set'));
+    command(editor);
+    await Waiter.pWait(10);
+    assert.isFalse(lastState.get().getOrDie('Should be set'));
+    McEditor.remove(editor);
+  };
+
+  context(`Should emit 'ToogleToolbarDrawer' event when toggled`, () => {
+    Arr.each<ToolbarMode>([ 'floating', 'sliding' ], (toolbarMode) => {
+      it(`TINY-9271: Emits 'ToggleToolbarDrawer' in ${toolbarMode} via execCommand`, async () => {
+        await pTestEvent(toolbarMode, (editor) => editor.execCommand('ToggleToolbarDrawer'));
+      });
+      it(`TINY-9271: Emits 'ToggleToolbarDrawer' in ${toolbarMode} via user click`, async () => {
```

Comment:
```suggestion

      it(`TINY-9271: Emits 'ToggleToolbarDrawer' in ${toolbarMode} via user click`, async () => {
```
Just some "fluff" that's normally included for readability 🤷

Reply:
Updated [here](https://github.com/tinymce/tinymce/pull/8238/commits/bf58856c776bbe02e4ba4b99159eea982121e23b)